web/crux/src/swagger/crux.dto.ts

```diff
@@ -32,14 +32,6 @@ import {
   VersionResponse,
```

Comment:
Okey for deleting the unused values from here, but we'll delete the whole file.

Reply:
I know, I just wanted to clean it up a bit.

Reply:
Okey.